From 533a8013d3bc651110e3fa4a85d97897ed9bcd2e Mon Sep 17 00:00:00 2001
From: Steven Rostedt <rostedt@goodmis.org>
Date: Wed, 5 Jul 2017 13:15:02 -0400
Subject: [PATCH 332/332] Linux 4.9.47-rt37 REBASE

Signed-off-by: Steven Rostedt <rostedt@goodmis.org>
---
 localversion-rt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/localversion-rt b/localversion-rt
index 9a218ca..a3b2408 100644
--- a/localversion-rt
+++ b/localversion-rt
@@ -1 +1 @@
--rt23
+-rt37
-- 
2.1.4

