From 78216b9200ec27983bbb64097aa3bc5770d81d71 Mon Sep 17 00:00:00 2001
From: Julia Cartwright <julia@ni.com>
Date: Mon, 15 Jan 2018 13:10:48 -0600
Subject: [PATCH 351/356] Linux 4.9.76-rt61 REBASE

---
 localversion-rt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/localversion-rt b/localversion-rt
index 9a218ca..9b7de93 100644
--- a/localversion-rt
+++ b/localversion-rt
@@ -1 +1 @@
--rt23
+-rt61
-- 
2.1.4

