From 7cbf5ab0e5248b1804529bf6da06048f069cc3c9 Mon Sep 17 00:00:00 2001
From: Clark Williams <williams@redhat.com>
Date: Wed, 19 Feb 2020 23:07:42 -0600
Subject: [PATCH 367/367] Linux 4.9.214-rt137 REBASE

Signed-off-by: Clark Williams <williams@redhat.com>
---
 localversion-rt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/localversion-rt b/localversion-rt
index 9a218ca23053..41b444e910ef 100644
--- a/localversion-rt
+++ b/localversion-rt
@@ -1 +1 @@
--rt23
+-rt137
-- 
2.28.0

