From bb73f5abb410b989451cad6facebedd98d8c4fc9 Mon Sep 17 00:00:00 2001
From: Julia Cartwright <julia@ni.com>
Date: Thu, 20 Dec 2018 14:25:00 -0600
Subject: [PATCH 367/368] Linux 4.9.178-rt131 REBASE

---
 localversion-rt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/localversion-rt b/localversion-rt
index 9a218ca..a328b97 100644
--- a/localversion-rt
+++ b/localversion-rt
@@ -1 +1 @@
--rt23
+-rt131
-- 
2.7.4

